coderd/templates.go

```diff
@@ -504,7 +503,7 @@ func (api *API) patchTemplateMeta(rw http.ResponseWriter, r *http.Request) {
 			DisplayName: displayName,
 			Description: desc,
 			Icon:        icon,
-			DefaultTtl:  int64(maxTTL),
+			DefaultTtl:  int64(defaultTTLMillis),
```

Comment:
Thought this had to be a bug but I was surprised to notice we're storing nanoseconds in the database... 😅